pkg/tinydtls/include/tinydtls/config.h

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Where is this header included? If it is not supposed to, but the defines are only used with `CFLAGS`, than I prefer to add the `tinydtls_config` group to `tinydtls`'s `doc.txt`.

Reply:
Yes, the defines are only used with `CFLAGS`.

> I prefer to add the `tinydtls_config` group to `tinydtls`'s `doc.txt`.

`tinydtls_config` group is already added to the `pkg_tinydtls` group and it shows up as modules of the group at the bottom of the page. Or do you mean putting all the defines here directly in the `doc.txt` and there is no need anymore for this file (`config.h`) then?

Reply:
Yes, I meant putting it into the `doc.txt`. Having a configuration header that is not included anywhere might be confusing.

Reply:
Done.